MozillaFoundation/foundation.mozilla.org
# Hyphenate taxonomies in libraries filters

# Description



Link to sample test page:
Related PRs/issues: #10154 

# Testing

1. Go to '/cms/snippets/wagtailpages/researchtopic/'
2. Change the title of an existing and used research topic to have a very long single word in the title. E.g. `Pseudopseudohypoparathyroidism for all words`
3. Go to '/en/research/library/'
4. See how the long word is broken up into two lines
